bin/generate_unfiltered_sce.R

```diff
@@ -125,6 +135,15 @@ unfiltered_sce <- unfiltered_sce |>
   add_gene_symbols(gene_info = gtf) |>
   scuttle::addPerFeatureQCMetrics()
 
+# read in sample metadata and filter to sample ids
+sample_metadata_df <- readr::read_tsv(opt$sample_metadata_file) |> 
+  # rename sample id column
+  dplyr::rename(sample_id = "scpca_sample_id")
```

Comment:
I think the renaming here is fine. I prefer to leave the scpcaTools function a bit more prescriptive & less specific to this project.

The quotes here aren't really needed, but I think it makes the funciton a bit more clear/consistent? I go back and forth, partly because I think this is one of the less well-designed functions in `dplyr`. Convenient, but confusing.
```suggestion
  dplyr::rename("sample_id" = "scpca_sample_id")
```